kite-data/kite-data-hbase/src/main/java/org/kitesdk/data/hbase/avro/io/ColumnEncoder.java

```diff
@@ -90,7 +90,7 @@ public void writeDouble(double d) throws IOException {
 
   @Override
   public void writeString(Utf8 utf8) throws IOException {
-    out.write(utf8.getBytes());
+    out.write(utf8.getBytes(), 0, utf8.getBytesLength());
```

Comment:
I think the Travis build failed because you have a typo. The method is called `getByteLength()` (no 's'). If you can fix that and update the PR, I can merge this in.